### PR TITLE
5164  serialize bulkOprInp_t (4-2-stable)

### PR DESCRIPTION
**Pull request by jasoncoposky**

### CODE REVIEW

File: scripts/irods/test/test_rule_engine_plugin_framework.py [hunk 1]
```diff
@@ -37,6 +38,49 @@ def setUp(self):
...
+
+        try:
+            lib.make_deep_local_tmp_dir(local_dir, depth, files_per_level, file_size)
+
+            logical_path = os.path.join(self.admin.session_collection, local_dir)
+
+            config = IrodsConfig()
+            core_re_path = os.path.join(config.core_re_directory, 'core.re')
+
+            print("XXXX - STARTn")
```
Comment (korydraughn) [inline scripts/irods/test/test_rule_engine_plugin_framework.py:60]: is this print statement from debugging?

Comment (korydraughn) [reply scripts/irods/test/test_rule_engine_plugin_framework.py:60]: there is also a trailing "n" on "START"

**Review by korydraughn — APPROVED** [review]
Looks good to me.

File: scripts/irods/test/test_rule_engine_plugin_framework.py [hunk 2]
```diff
@@ -37,6 +38,49 @@ def setUp(self):
...
+        if(*err < 0) {{
+            writeLine("serverLog", "XXXX - *err [*msg]")
+        }}
+    }}
+
+                    """.format(logical_path))
+
+                self.admin.assert_icommand(['iput', '-b', '-r', local_dir], 'STDOUT_SINGLELINE', 'Running')
+                self.admin.assert_icommand(['ils', '-l'], 'STDOUT_SINGLELINE', 'rods')
+                self.admin.assert_icommand(['imeta', 'ls', '-C', logical_path], 'STDOUT_SINGLELINE', 'logical_path')
```
Comment (trel) [inline scripts/irods/test/test_rule_engine_plugin_framework.py:79]: last logical_path should not have single quotes?

**Review by trel — COMMENTED** [review]
only see the extra debugging code - rest seems good

File: scripts/irods/test/test_rule_engine_plugin_framework.py (hunk 1 above)
Comment (jasoncoposky) [reply scripts/irods/test/test_rule_engine_plugin_framework.py:60]: Cruft.

**Comment by trel** [comment]
change to 5164 in the commit message

**Comment by jasoncoposky** [comment]
done.

**Comment by trel** [comment]
yeps, let's # it.

**Pull request merged**